From 7f0cd4b6b56183b0afbefd01425e5ebd2b8733b4 Mon Sep 17 00:00:00 2001
From: Martin Jansa <martin.jansa@gmail.com>
Date: Mon, 8 Jul 2024 13:18:11 +0200
Subject: [PATCH] Makefile: fix typo in soname argument

* introduced in:
  https://sourceforge.net/p/giflib/code/ci/b65c7ac2905c0842e7977a7b51d83af4486ca7b8/
  there is no LIBUTILMAJOR variable only LIBUTILSOMAJOR leading to:

  ld: fatal error: -soname: must take a non-empty argument
  collect2: error: ld returned 1 exit status

  with some linkers like GOLD

Signed-off-by: Martin Jansa <martin.jansa@gmail.com>
---
Upstream-Status: Submitted [https://sourceforge.net/p/giflib/code/merge-requests/17/]

 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 87966a9..41b149e 100644
--- a/Makefile
+++ b/Makefile
@@ -109,7 +109,7 @@ $(LIBUTILSO): $(UOBJECTS) $(UHEADERS)
 ifeq ($(UNAME), Darwin)
 	$(CC) $(CFLAGS) -dynamiclib -current_version $(LIBVER) $(OBJECTS) -o $(LIBUTILSO)
 else
-	$(CC) $(CFLAGS) -shared $(LDFLAGS) -Wl,-soname -Wl,$(LIBUTILMAJOR) -o $(LIBUTILSO) $(UOBJECTS)
+	$(CC) $(CFLAGS) -shared $(LDFLAGS) -Wl,-soname -Wl,$(LIBUTILSOMAJOR) -o $(LIBUTILSO) $(UOBJECTS)
 endif
 
 libutil.a: $(UOBJECTS) $(UHEADERS)
